src/Fabulous.Avalonia/Widgets/Controls/Button.fs

```diff
@@ -37,6 +37,9 @@ module ButtonBuilders =
         static member inline Button<'msg>(text: string, onClicked: 'msg) =
             WidgetBuilder<'msg, IFabButton>(Button.WidgetKey, ContentControl.ContentString.WithValue(text), Button.Clicked.WithValue(fun _ -> box onClicked))
 
+        static member inline Button<'msg>(onClicked: 'msg) =
```

Comment:
I feel like we should not allow users to create empty button, even if it's allowed by Avalonia.
Instead I would recommend using the `Button(msg, contentWidget)` instead.

Like for example, instead of an empty button with just a background color:
```fs
Button(KillCell)
    .background(SolidColorBrush(Color.Red))
```

You could do a button with a rectangle inside:
```fs
Button(KillCell,
    Rectangle()
        .fill(SolidColorBrush(Color.Red))
)
```

It's longer but the API ergonomic feels better.
What do you think?